lib/index.js

```diff
@@ -175,16 +190,23 @@ export class Baker extends EventEmitter {
     this.getStaticPath = staticBlock;
     this.nunjucks.getStaticPath = staticBlock;
 
-    // hook up our custom static tag and pass in the manifest
-    this.nunjucks.addCustomTag('static', staticBlock);
-
     const staticAbsoluteBlock = createStaticAbsoluteBlock(
       this.input,
       this.domain,
       this.pathPrefix,
       [this.assets, this.sass]
     );
 
+    // hook up our custom static tag and pass in the manifest
+    this.nunjucks.addCustomTag(
+      'static',
+      isProductionEnv &&
```

Comment:
Can potentially refactor this depending on the answer to comment above on line175

Reply:
this is different from line 175. This deals with js files. 175 handles other assets